bridges/primitives/runtime/src/chain.rs

```diff
@@ -199,7 +199,7 @@ pub trait Chain: Send + Sync + 'static {
 }
 
 /// A trait that provides the type of the underlying chain.
-pub trait UnderlyingChainProvider {
+pub trait UnderlyingChainProvider: Send + Sync + 'static {
```

Comment:
@svyatonik just out of curiosity, what exactly was the reason for adding `Send + Sync + 'static`?

Reply:
That's to avoid duplicating those lines:
```rust
	<MB as MessageBridge>::BridgedChain: Send + Sync + 'static,
	<MB as MessageBridge>::ThisChain: Send + Sync + 'static,
```
Which in turn are required, because `Chain` needs to be `Send + Sync + 'static` and when we use `UnderlyingChainProvider` we are implementing all `Chain` associated types, but we can't guarantee that it'll be `Send + Sync + 'static`. Alternative would be to `impl<T: UnderlyingChainProvider + Send + Sync + 'statuc> Chain for T { ... }`, but imo it is better

Reply:
aha, ok, thank you for explanation :)